src/states/node.rs

```diff
@@ -1593,6 +1616,72 @@ impl Node {
         result.with_value(Ok(()))
     }
 
+    fn handle_rt_req(&mut self,
+                     msg_id: MessageId,
+                     digest: sha256::Digest,
+                     src: Authority<XorName>,
+                     dst: Authority<XorName>)
+                     -> Evented<Result<(), RoutingError>> {
+        // TODO: Compare the hash and don't send anything if he requester is up-to-date.
+        let mut result = Evented::empty();
+        let sections = self.peer_mgr.pub_ids_by_section(self.full_id.public_id());
```

Comment:
`let sections = ...` is also duplicated. But if you think it's not worth trying to deduplicate that, I won't fight about it ;)